bundlesize.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": [
+    {
+      "path": "./dist/ui-kit.esm.js",
+      "maxSize": "175 kB"
```

Comment:
Hmm are you sure this is correct?

https://bundlephobia.com/result?p=@commercetools-frontend/ui-kit@9.9.0

Reply:
Can you give more information? What is correct? The max size?

Reply:
Yes the max size...currently it's like >700KB

Reply:
No, it's correct. When I run bundlesize it gives ~165.

Reply:
![Screen Shot 2019-07-04 at 11 40 34 AM](https://user-images.githubusercontent.com/2425013/60656856-8f48dc80-9e50-11e9-91ed-fc48902b7b69.png)
which is very close to your links result...

Reply:
Ah ok it gzippes it

<img width="643" alt="image" src="https://user-images.githubusercontent.com/1110551/60657169-201fb800-9e51-11e9-8285-d99d74ddd8c1.png">

Reply:
In fact I think we should lower this. It should be maybe 2-5%?